Packs/Vectra_AI/Integrations/VectraAIEventCollector/VectraAIEventCollector.py

```diff
@@ -129,28 +129,32 @@ def add_parsing_rules(event: Dict[str, Any]) -> Any:
         - Event `Dict[str, Any]` with the added Parsing Rules or skip.
     """
 
-    parsing_rules_to_add = ["_time"]
-
     try:
         # Process detection
         if DETECTION_TIMESTAMP_KEY in event:
-            event[parsing_rules_to_add[0]] = datetime.strptime(
-                event.get(DETECTION_TIMESTAMP_KEY), DETECTION_TIMESTAMP_FORMAT  # type: ignore
-            ).strftime(XSIAM_TIME_FORMAT)
+            event[XSIAM_PARSING_RULES[0]] = timestamp_to_datestring(
```

Comment:
Why is XSIAM_PARSING_RULES a list? If you are aiming for a generic approach it is better to have a dict with the XSIAM key and the response key.

Reply:
It's true, will change to `Dict[str]`.